mysql-test/suite/tokudb/t/type_date.test

```diff
@@ -180,52 +180,6 @@ EXPLAIN SELECT COUNT(*) FROM t1 WHERE a = '2015-12-08 02:00:09';
 
 DROP TABLE t1;
 
-#
-# Bug#28687: Search fails on '0000-00-00' date after sql_mode change
```

Comment:
What happened to this test snippet in the core server?

Reply:
I've changed it to match it